packages/routing-utils/src/schemas.ts

```diff
@@ -95,6 +95,9 @@ export const routesSchema = {
               maxLength: 32,
             },
           },
+          caseSensitive: {
+            type: 'boolean',
+          },
```

Comment:
Is this referring to the `src` or `dest` being case sensitive? I assume src but not sure if there is a spec for this new feature. Also I don't see tests for confirming it works?

Reply:
Tests were added in the linked proxy PR as we can't test the routing logic here, a test to ensure this validates against the schema was added in this PR though. This is for `src` being case-sensitive It's not exactly a new feature but a different way to toggle an existing feature as we currently treat `src` as case-insensitive when only routes are used but treat it as case-sensitive when `rewrites`, `redirects`, or `headers` are present. 

This allows us to treat specific routes as case-sensitive in Next.js so we can gradually migrate to case-sensitive routes.